sql/dql/intro-joins/relative-database-intro.md

```diff
@@ -0,0 +1,108 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - joins
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+---
+
+# Relational Databases
+
+---
+
+## Content
+
+You are now familiar with querying individual tables.
+
+Real databases are often comprised of more than one table.
+
+Relational databases consist of tables and entries that connected to one another in some way. We call this connection a *relationship*.
+
+Consider the `pokedex` database, it contains two tables:
+
+`pokemon_health`
+| id | name       | health |
+|----|------------|--------|
+| 1  | Venusaur   | 42     |
+| 2  | Charmander | 35     |
+
+`pokemon_attack`
+| id | type   | base-attack |
+|----|--------|-------------|
+| 1  | poison | 21          |
+| 2  | fire   | 15          |
```

Comment:
Tables are displayed wrong / aren't rendered as tables at all within the app. Maybe adding a space after `pokemon health` and `pokemon_attack` will fix this

Reply:
yes :ok_hand: